src/frontend/App.vue

```diff
@@ -1,32 +1,27 @@
 <template>
-    <main>
-        <nav-bar />
+    <main>        
         <router-view />
     </main>
 </template>
 
-<script>
-    import NavBar from './components/NavBar.vue';
-
-    export default {
-        components: {
-            'nav-bar': NavBar
-        }
-    }
+<script>    
 </script>
 
 <style lang="scss">
+    @import url(https://fonts.googleapis.com/css?family=Arimo|DM+Serif+Display|Poppins:700&display=swap);
+
     * {
         font-family: sans-serif;
         box-sizing: border-box;
     }
 
-    body {
+    html, body {
         padding: 0;
         margin: 0;
+        height: 100%;
     }
 
-    article {
-        text-align: center;
+    main{
+        height: 100%;
     }
```

Comment:
mir ist noch aufgefallen dass es aktuell nicht wirklich auf desktop nutzbar ist.
Eine einfache lösung könnte sein mit dem main element folgenes zu machen:

```suggestion
        margin: 0 auto;
        max-width: 500px;
    }
```

<img width="999" alt="Bildschirmfoto 2019-10-25 um 22 28 22" src="https://user-images.githubusercontent.com/9198250/67602399-c54efb00-f776-11e9-9352-2b00a6dd67a3.png">

Reply:
Wird genauso übernommen.

Reply:
nice 👍